tutorials/source-modeling/plot_beamformer_lcmv.py

```diff
@@ -0,0 +1,207 @@
+# ``None``.
+#
+# Furthermore, we will optimize the orientation of the sources such that output
+# power is maximized. This is achieved by setting ``pick_ori='max-power'``.
+# This gives us one source estimate per source (i.e., voxel), which is known
+# as a scalar beamformer. It is also possible to compute a vector beamformer,
+# which gives back three estimates per voxel, corresponding to the three
+# directions of the source. This can be achieved by setting
+# ``pick_ori='vector'`` and will yield a :class:`volume vector source estimate
+# <mne.VolVectorSourceEstimate>`.
+
+filters = make_lcmv(evoked.info, forward, data_cov, reg=0.05,
+                    noise_cov=noise_cov, pick_ori='max-power',
+                    weight_norm='unit-noise-gain', rank=None)
+
+# You can save the filter for later use with:
+# filters.save('filters-lcmv.h5')
+
+###############################################################################
+# Apply the spatial filter
+# ------------------------
+# The spatial filter can be applied to different data types: raw, epochs,
+# evoked data or the data covariance matrix to gain a static image of power.
+# The function to apply the spatial filter to :class:`~mne.Evoked` data is
+# :func:`~mne.beamformer.apply_lcmv` which is
+# what we will use here. The other functions are
+# :func:`~mne.beamformer.apply_lcmv_raw`,
+# :func:`~mne.beamformer.apply_lcmv_epochs`, and
+# :func:`~mne.beamformer.apply_lcmv_cov`.
+
+stc = apply_lcmv(evoked, filters, max_ori_out='signed')
+
+###############################################################################
+# Visualize the reconstructed source activity
+# -------------------------------------------
+# We can visualize the source estimate in different ways, e.g. as an overlay
+# onto the MRI or as a glass brain.
+# The plots show brain activity in the right temporal lobe around 100 ms post
+# stimulus. This is coherent with the left-ear auditory stimulation of the
+# experiment.
+
+lims = [0.3, 0.45, 0.6]
+
+stc.plot(
+    src=forward['src'], subject='sample', subjects_dir=subjects_dir,
+    clim=dict(kind='value', pos_lims=lims), mode='stat_map',
+    initial_time=0.087, verbose=True)
+###############################################################################
+stc.plot(
+    src=forward['src'], subject='sample', subjects_dir=subjects_dir,
+    mode='glass_brain', clim=dict(kind='value', lims=lims),
+    initial_time=0.087, verbose=True)
+
+
+###############################################################################
+# References
+# ----------
+# .. [1] Van Veen et al. Localization of brain electrical activity via
+#        linearly constrained minimum variance spatial filtering.
+#        Biomedical Engineering (1997) vol. 44 (9) pp. 867--880
+#
+# .. [2] Gross et al. (2001) Dynamic imaging of coherent sources: Studying
+#        neural interactions in the human brain.
+#        PNAS vol. 98 (2) pp. 694-699. https://doi.org/10.1073/pnas.98.2.694
```

Comment:
This is the old sad way of doing things. Nowadays you just want to use `footbibliography`, see for example `tutorials/preprocessing/plot_60_maxwell_filtering_sss.py`

Reply:
@larsoner I will check the overlap and report back and also fix the references. Travis failure must be unrelated?

Reply:
Yes, OSX continues to timeout